renderer/plugin.js

```diff
@@ -67,6 +66,16 @@ org.ekstep.questionunitReorder.RendererPlugin = org.ekstep.contentrenderer.quest
   postQuestionShow: function (event) { // eslint-disable-line no-unused-vars
     ReorderingController.question = this._question; // eslint-disable-line no-undef
     QSTelemetryLogger.logEvent(QSTelemetryLogger.EVENT_TYPES.ASSESS); // eslint-disable-line no-undef
+    // ReorderingController.renderQuestion();
+    if (!this._question.state) {
+      this._userWords = [];
+    } else {
+      this._userWords = [];
+      var temp = this._question.state.val;
+      _.each(temp, function (v, i) {
+        ReorderingController.wordClick(v.id);

```

Comment:
WordClick(v.id) will log telemetry events. It should not log while updating the state.